client-app/components/atoms/alert/alert.vue

```diff
@@ -1,6 +1,84 @@
 <template>
-  <div class="flex items-center space-x-2 px-3 py-3 bg-red-100 rounded-sm">
-    <i class="fa fa-times-circle text-red-500" aria-hidden="true"></i>
-    <span><slot></slot></span>
+  <div class="vc-alert" :class="[type && `vc-alert--${type}`, text && 'vc-alert--text']">
+    <i v-if="icon" :class="iconClasses" class="vc-alert__icon" aria-hidden="true" />
+    <div class="vc-alert__content">
+      <slot />
+    </div>
   </div>
 </template>
+
+<script setup lang="ts">
+import { computed, PropType } from "vue";
+
+const props = defineProps({
+  text: Boolean,
```

Comment:
It's too strange to make boolean `text` property. Maybe it's better to make `text` one of `type` enum.